### DISCUSSION TITLE
internet.js

### DISCUSSION THREAD

**Discussion by ghost** [TurboWarp/extensions#862] [Show and tell]
a internet extension

http://scratchextensions.7m.pl/extensions/internet.js

(run without sandbox)

**Comment by BlueToadMakerr** [comment]
Its not working. I get Uncaught SyntaxError: Unexpected token ':'

**Comment by ThatBeaverDev** [comment]
you need to make a pull request if you want this added to turbowarp officially

**Reply by LilyMakesThings** [reply]
you do not need to tell them this many times

**Reply by ThatBeaverDev** [reply]
Sorry, won't do in the future
